### PR TITLE
Replace workerpool

### PR DESCRIPTION
**Pull request by emensch**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] All applicable changes have been documented
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment

### CODE REVIEW

File: packages/c2pa/src/lib/pool/error.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+interface SerializedError {
```
Comment (dkozma) [inline packages/c2pa/src/lib/pool/error.ts:10]: Reminder to add copyright headers

File: packages/c2pa/src/lib/poolWrapper.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  dispose: () => void;
+}
+
+export async function createPoolWrapper(config: WorkerPoolConfig): Promise<SdkWorkerPool> {
+  const res = await fetch(config.scriptSrc);
+
+  if (!res.ok) throw new InvalidWorkerSourceError(config.scriptSrc, res);
+
+  const src = await res.text();
+    // @TODO: check subresource integrity
```
Comment (dkozma) [inline packages/c2pa/src/lib/poolWrapper.ts:27]: 👍🏼

Comment (dkozma) [reply packages/c2pa/src/lib/poolWrapper.ts:27]: @emensch Can you create an issue for this so we can track?

**Review by dkozma — APPROVED** [review]
@emensch This looks great, however can we publish this as a prerelease while we do some QE on this change and add some testing around it?

File: packages/c2pa/src/lib/poolWrapper.ts (hunk 2 above)
Comment (emensch) [reply packages/c2pa/src/lib/poolWrapper.ts:27]: Done https://github.com/contentauth/c2pa-js/issues/84

**Comment by emensch** [comment]
Released as `c2pa@0.14.0-rc.0`

**Pull request merged**